.github/workflows/test-private.yaml

```diff
@@ -34,7 +34,9 @@ jobs:
       fail-fast: false
       matrix:
         chart:
-          - "phylogatr-web"
+          # TODO: Add back once phylogatr is fixed
```

Comment:
May need to rebase against main branch as these changes already happened in main branch.

Reply:
I did `git rebase main` with local `main` being fully up-to-date and pushing.  If you want to pull you'll first have to likely do forced pull.